maps/src/MapTODPointing.cxx

```diff
@@ -0,0 +1,169 @@
+#include <pybindings.h>
+#ifdef OPENMP_FOUND
+#include <omp.h>
+#endif
+
+#include <G3Module.h>
+#include <G3Timestream.h>
+#include <G3Quat.h>
+#include <G3Data.h>
+#include <G3Map.h>
+#include <G3Units.h>
+#include <maps/G3SkyMap.h>
+#include <maps/pointing.h>
+#include <calibration/BoloProperties.h>
+
+class MapTODPointing : public G3Module {
+public:
+	MapTODPointing(std::string pointing, std::string timestreams,
+	    G3SkyMapConstPtr stub_map, std::string tod_pointing,
+	    std::string bolo_properties_name);
+	virtual ~MapTODPointing() {}
+
+	void Process(G3FramePtr frame, std::deque<G3FramePtr> &out);
+
+private:
+	std::string pointing_;
+	std::string timestreams_;
+
+	G3SkyMapConstPtr stub_;
+
+	std::string output_;
+	std::string boloprops_name_;
+
+	BolometerPropertiesMapConstPtr boloprops_;
+
+	SET_LOGGER("MapTODPointing");
+};
+
+EXPORT_G3MODULE("maps", MapTODPointing,
+    (init<std::string, std::string, G3SkyMapConstPtr, std::string, std::string>
+     ((arg("pointing"), arg("timestreams"), arg("stub_map"), arg("tod_pointing"),
+     arg("bolo_properties_name")="BolometerProperties"))),
+"MapTODPointing(pointing, timestreams, stub_map, tod_pointing, bolo_properties_name=\"BolometerProperties\")\n"
+"\n"
+"Compute pixel pointing for timestreams for a map with properties (projection, etc.) specified\n"
+"by the stub_map argument. \n\n"
```

Comment:
I would add a comment here about what this is *for*. I.e. "The outputs of this module are identical to internally-calculated quantities in the map maker and can be used for debugging map-making or studying individual-detector pixel coverage. The outputs are not consumed by any stage of the map-making pipeline and exist to be read by humans."